kvmagent/kvmagent/plugins/ovn.py

```
@@ -291,7 +290,41 @@ def stop_ovn_service(self, req):
+            targetDriver = cmd.nicNameDriverMap.__dict__[nicName]
+            if driver == "mlx5_core" or driver == targetDriver:
+                # mellanox nic(like cx-5) does not need vfio driver
+                continue
+
+            # for nested vm, the driver is should be uio_pci_generic
+            r, _, e = bash.bash_roe(ovn.DevBindBin + " -b {driver} {pciAddress}"
+                                    .format(driver=targetDriver, pciAddress=pciAddress))
+            if r != 0:
+                logger.debug(
+                    "change change nic [pci address: {}] driver to {} failed: {}}"
+                    .format(pciAddress, targetDriver, e))
```

Ruff (0.8.2)

`.format` call has invalid format string: Single '}' encountered in format string

(F521)